sw/device/tests/sim_dv/sleep_pin_wake_test.c

```diff
@@ -91,6 +117,11 @@ bool test_main(void) {
                                             kDifPinmuxPadKindMio, in_attr,
                                             &out_attr));
 
+    // This print is placed here on purpose.
+    // sv sequence is waiting for this print log followed by
+    // Pad Section. So do not put any other print between
+    // this LOG_INFO and LOG_INFO("Pad Selection");
+    LOG_INFO("pinmux_init end");
```

Comment:
I assume this is already included in the sv side?

Reply:
yes in [here](https://cs.opensource.google/opentitan/opentitan/+/master:hw/top_earlgrey/dv/env/seq_lib/chip_sw_sleep_pin_wake_vseq.sv;drc=0a914d2290a3ba4284c3af9a537c5872cd793721;l=53)